<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>338</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>338</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_photo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>上传头像</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>55</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-width:0;
border-style:outset;
font: 14pt &quot;Consolas&quot;;
</string>
   </property>
   <property name="text">
    <string>saufhus</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_photo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>37</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">min-width:60px;
min-height:60px;
max-width:60px;
max-height:60px;
border-radius:30px;
border:1px solid black;

</string>
   </property>
   <property name="text">
    <string>头像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>680</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-width:0;
border-style:outset

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>680</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-width:0;
border-style:outset
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_zod">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>505</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 1px;border-style: solid;border-color: rgb(255, 255, 255);
font: 12pt &quot;Consolas&quot;;
</string>
   </property>
   <property name="text">
    <string>星座</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_age">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>445</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 1px;border-style: solid;border-color: rgb(255, 255, 255);
font: 12pt &quot;Consolas&quot;;
</string>
   </property>
   <property name="text">
    <string>年龄</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="edit_signa">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>271</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-width:0;
border-style:outset;
font: 12pt &quot;Consolas&quot;;

</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Consolas'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun'; font-size:9pt;&quot;&gt;交个朋友吧~&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_sex">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>335</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(218, 218, 218);
background-color: rgb(255, 255, 255);
font: 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
   <property name="minimumContentsLength">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>男</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>女</string>
    </property>
   </item>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>385</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Consolas&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_city">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>560</y>
     <width>161</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 12pt &quot;Consolas&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>338</width>
     <height>768</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/base/设置界面图.jpg</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>edit_name</zorder>
  <zorder>label_photo</zorder>
  <zorder>btn_ok</zorder>
  <zorder>btn_cancel</zorder>
  <zorder>label_zod</zorder>
  <zorder>label_age</zorder>
  <zorder>edit_signa</zorder>
  <zorder>cbox_sex</zorder>
  <zorder>dateEdit</zorder>
  <zorder>cbox_city</zorder>
  <zorder>btn_photo</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
